_15thnight/api/account.py

```diff
@@ -14,26 +15,26 @@ def login():
     Authenticate with the application.
     """
     # TODO: issue API key here instead of cookie
-    form = LoginForm(request.form)
+    form = LoginForm(request.json_multidict)
     if form.validate_on_submit():
-        user = User.get_by_email(request.form['email'].lower())
-        password = request.form.get('password')
+        user = User.get_by_email(form.email.data.lower())
+        password = form.password.data
         if user is not None and user.check_password(password):
-            session['logged_in'] = True
             login_user(user)
-            return '', 200
+            return jsonify(user)
         return jsonify(error='Invalid username/password.', _status_code=401)
     return jsonify(error='Invalid form data', _status_code=400)
 
 
 @account_api.route('/logout', methods=['POST'])
+@login_required
 def logout():
     """
     Deauthenticate with the application.
     """
     # TODO: de-auth API key
```

Comment:
The API keys will only be needed if we ever did some kind of other integration outside of the browser like a mobile app.